python/lsst/ts/pmd/csc.py

```diff
@@ -118,6 +118,7 @@ async def telemetry(self):
             breakpoint
             while True:
                 position = await self.component.get_slots_position()
+                self.log.debug(f"{position=}")
```

Comment:
I suggest not logging information that is published as an event; it needlessly duplicates information on the SAL network. I would ditch both this log message and the next one.